src/main/resources/application.yaml

```diff
@@ -15,6 +15,52 @@ management:
 spring:
   application:
     name: Spring Boot Template
+
+security:
+  anonymousPaths:
+    - "/"
+    - "/health"
+    - "/health/liveness"
+    - "/actuator/**"
+    - "/loggers/**"
+    - "/swagger-ui.html"
+    - "/swagger-resources/**"
+    - "/v2/api-docs"
+    - "/webjars/springfox-swagger-ui/**"
+    - "/search/**"
+    - "/csrf"
+    - "/error"
+    - "/favicon.ico"
+  ### Update list with service id of any service going to interact with casse document api
+  authorisedServices:
+    - ccd-case-document-am-api
+    - ccd_gw
+    - xui_webapp
+
+
+
+### dependency configuration
+auth.provider.service.client.baseUrl: ${IDAM_S2S_URL:http://localhost:4502}
+idam:
+  s2s-auth:
+    totp_secret: ${S2S_SECRET:AAAAAAAAAAAAAAAB}
+    microservice: ccd-case-document-am-api
+    url: ${S2S_URL:http://localhost:4502}
+    pbaFromUserAccountMap: false
```

Comment:
what is the purpose of this variable? can we please remove if it is not required.

Reply:
why I asked, pba is internal to ref-data app and this variable is also set to false. so not sure it hinders the our api request. can you please check the dependency on it.

Reply:
yeah I checked this and there is no use related to security.I have removed this.